src/styles/blocks/products.scss

```
@@ -0,0 +1,11 @@
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: getProductsWidth($card-width, $cards-num, $card-margin);
+
+  &__product-card {
+    display: flex;
+    margin-bottom: 50px;
+  }
```

display property should be added to the main card styles (not to mix class)

```suggestion
  &__product-card {
    margin-bottom: 50px;
  }
```